opencensus/trace_exemplar_quickstart/main.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2020 Google LLC
```

Comment:
Please move this to the `opencensus` directory directly. Not a quickstart anymore, so it doesn't need to be in another directory.

Reply:
Done